jdaviz/configs/default/plugins/model_fitting/tests/test_fitting.py

```diff
@@ -25,6 +26,22 @@ def build_spectrum(sigma=0.1):
     return x, y + noise
 
 
+def test_model_params():
+    # expected model parameters:
+    model_parameters = {"Gaussian1D": ["amplitude", "stddev", "mean"],
+                        "Const1D": ["amplitude"],
+                        "Linear1D": ["slope", "intercept"],
+                        "PowerLaw1D": ["amplitude", "x_0", "alpha"],
+                        "Lorentz1D": ["amplitude", "x_0", "fwhm"],
+                        "Voigt1D": ["x_0", "amplitude_L", "fwhm_L", "fwhm_G"],
+                        }
+
```

Comment:
Maybe something this like would let us catch if model listing becomes outdated?

```python
assert sorted(model_parameters.keys()) == sorted(MODELS.keys())
```

Reply:
I did something similar in bfe2be7d4df38f0794737f8f133755781bbf2c02 - let me know if you think we need to cover anything else.

Reply:
LGTM. Thanks!